app/src/main/java/com/example/minwoo/pingpongchat_client/LoginActivity.java

```diff
@@ -74,18 +118,29 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
                 String personName = account.getDisplayName();
                 String personEmail = account.getEmail();
                 String personId = account.getId();
+                String personIdToken = account.getIdToken();
 
                 Log.e("GoogleLogin", "personName=" + personName);
```

Comment:
Log.e is error logging 
better use other log option
preferernce: https://hashcode.co.kr/questions/1007/%EC%95%88%EB%93%9C%EB%A1%9C%EC%9D%B4%EB%93%9C%EC%97%90%EC%84%9C-logvlogdlogilogwloge-%EA%B0%81%EA%B0%81-%EC%96%B8%EC%A0%9C-%EC%93%B0%EB%82%98%EC%9A%94